src/index.spec.js

```diff
@@ -14,7 +14,6 @@ describe('babel plugin', () => {
   //       to starting subprocesses.
   var procPool
   beforeEach(() => { procPool = new ProcessPool })
-  afterEach(() => { procPool.destroy() })
```

Comment:
That line is important. Without it, if you're developing `sigh-babel` with `sigh -w` then it will leak processes and eventually fail. The tests work for me using the line, also on the CI server. Do they only fail for you after your change or before it as well? If the latter then it could be caused by the change, if the former it could represent another issue.